src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -129,11 +129,6 @@ std::string hex_data(const unsigned char* const buffer,
 
 void UsbConnection::OnInTransfer(libusb_transfer* transfer) {
   SDL_LOG_AUTO_TRACE();
-  if (transfer->status == LIBUSB_TRANSFER_CANCELLED) {
```

Comment:
@RomanReznichenkoLuxoft maybe you should just one more check here?
I see that upon disconnect there are flags like `disconnecting_`, `waiting_out_transfer_cancel_`, `waiting_in_transfer_cancel_` 
In this case I guess you should add something like:
```
if (!waiting_in_transfer_cancel_ && transfer->status == LIBUSB_TRANSFER_CANCELLED) {
```

And probably the same check for out transfer

Reply:
After replacing this condition, the code does not work in the if

Reply:
hope it works )